src/client/styles/main.scss

```diff
@@ -170,3 +170,66 @@ input[type=submit]:focus {
   outline-style: solid;
   outline-width: 3px;
 }
+
+.nypl-results-list {
```

Comment:
This implementation is in the toolkit but it's for the hours table. The css classes could be updated for a more generic responsive table.